post-types/profile-pages.php

```diff
@@ -63,7 +63,15 @@ public function register_post_type()
 
         public function add_meta_box()
         {
-            // Profile settings
+          // Profile settings
+          add_meta_box(
```

Comment:
@Huyeng Here I add another meta box with the dropdown box for selecting the layout